### DISCUSSION TITLE
[@types/mixpanel-browser] Property 'track_pageview' does not exist on type 'OverridedMixpanel'. - mixpanel-browser v2.47.0

### DISCUSSION THREAD

**Discussion by beforeoafterm** [DefinitelyTyped/DefinitelyTyped#65666] [Issues with a @types package] [answered]
Hi, all!

I'm currently using React (`^18.1.0`) with TypeScript (`^5.0.4`) and mixpanel-browser (`^2.47.0`).

Using the new `track_pageview()` method in the latest release, however, I am encountering this TS error:

```
Property 'track_pageview' does not exist on type 'OverridedMixpanel'.
```

<img width="976" alt="Screenshot 2023-05-31 at 1 11 30 PM" src="https://github.com/mixpanel/mixpanel-js/assets/22500028/7e7ff2a0-c295-4778-8aa6-d1ea9d0296bb">

When I checked `mixpanel/index.d.ts`, it looks like the `Mixpanel` interface does not have the method signature defined for `track_pageview()`.

<img width="849" alt="Screenshot 2023-05-31 at 1 14 42 PM" src="https://github.com/mixpanel/mixpanel-js/assets/22500028/0ff86ffd-3704-44a1-a292-bf7ec9caac2a">

I believe we can fix this issue for TS users by adding the method signature:

```
track_pageview(properties?: Dict): void
```

There may also be a need to properly type the rest of the new functions and `Config` options from the [latest release of the `mixpanel-browser` library](https://github.com/mixpanel/mixpanel-js/releases/tag/v2.47.0).

[a comment by an automated account is not shown]

**Comment by wsfuller** [comment, marked as answer]
Having the same issue, have put up the following PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65672 to address your issue and the one I was having both about `track_pageview`

**Reply by wsfuller** [reply]
Related Discussion #65514
